src/components/CmsZone.vue

```diff
@@ -76,6 +72,7 @@
           :zone-id="zoneId"
         />
       </div>
+      <cms-observer v-if="isScrolling" :options="{ rootMargin: '25%' }" @intersect="next" />
```

Comment:
Why set the `rootMargin` to a percentage? I don't think we should be changing how far in advance we're loading based on the user's screen size?

Reply:
I thought it would lead to a smoother scrolling experience regardless of screen size. We'd already need to load more than one page on large screen. Happy to change this to an absolute amount instead. Any suggestions?